tools/clusters-checker/pkg/checkers/factory.go

```diff
@@ -0,0 +1,34 @@
+package checkers
+
+import (
+	"fmt"
```

Comment:
sort imports, there should be a space after "fmt"

Reply:
done.